js/parser.js

```diff
@@ -92,7 +92,9 @@ var ERMrest = (function(module) {
                     context.paging.row = {};
                     var row = modifierPath.match(/@before\(([^\)]*)\)/)[1].split(",");
                     for (var i = 0; i < context.sort.length; i++) {
-                        context.paging.row[context.sort[i].column] = decodeURIComponent(row[i]);
+                        // ::null:: to null, empty string to "", otherwise decode value
+                        var value = (row[i] === "::null::" ? null : (row[i] === "" ? "" : decodeURIComponent(row[i])));
```

Comment:
Is this test of empty string (and similar one below) necessary?  Won't `decodeURIComponent("")` return `""` already?